script/foreman-admin

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env ruby
+# -*- encoding: utf-8 -*-
+
+require 'rubygems'
+require 'clamp'
```

Comment:
> Why would someone install Foreman without using the installer? Personally, I feel like that's on them to resolve dependencies if they don't use the installer.

Installer's just a bit of extra configuration, packages still enforce dependencies.  You can install from packages or source, works fine.

Perhaps add the dependency to our packages (in which case for RPMs, this would use SCL) or use the existing bundler configuration?